### PR TITLE
Border ordering

### PR DESCRIPTION
**Pull request by madflow**
This is a XLSX specific fix for #271.

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @madflow has signed the CLA. Thanks for the pull request!

File: src/Spout/Writer/XLSX/Helper/StyleHelper.php [hunk 1]
```diff
@@ -110,13 +110,21 @@ protected function getBordersSectionContent()
             $border = $style->getBorder();
             if ($border) {
                 $content .= '<border>';
-                foreach ($border->getParts() as $part) {
-                    /** @var $part \Box\Spout\Writer\Style\BorderPart */
-                    $content .= BorderHelper::serializeBorderPart($part);
+
+                $sortOrder = ['left', 'right', 'top', 'bottom'];
```
Comment (adrilo) [inline src/Spout/Writer/XLSX/Helper/StyleHelper.php]: Can you add a comment saying why they need to be in this order (or just add the link to the issue)

**Comment by madflow** [comment]
- [x] Added a link to the issue

**Comment by adrilo** [comment]
👍

**Pull request merged**